src/main/java/sirius/pasta/noodle/SiriusClassAliasProvider.java

```diff
@@ -62,11 +64,12 @@ public void collectAliases(BiConsumer<String, Class<?>> consumer) {
         consumer.accept("CallContext", CallContext.class);
         consumer.accept("UserContext", UserContext.class);
         consumer.accept("WebContext", WebContext.class);
-        consumer.accept("Product", Product.class);
         consumer.accept("Injector", Injector.class);
         consumer.accept("Strings", Strings.class);
         consumer.accept("Files", Files.class);
         consumer.accept("Hasher", Hasher.class);
         consumer.accept("Amount", Amount.class);
+        consumer.accept("ContentHelper", ContentHelper.class);
```

Comment:
i'd replace the missing methods provided by this class either by static versions or even better by macros and eventually deprecate this helper.